brandonrhayes/Minimizing_Vaccination_Waste
# Results from investigation into assigning variables for inputs

Here's whats new:

- Added a function to PrimaryController.java file to initialize. What this means is that when I create a variable in the controller, I can assign that variable in SceneBuilder using the fx:id.
- Created a variable in PrimaryController.java for the first textfield input, the number of doses for a vial type. Variable name is "num_dose_1", subject to change after a naming convention is defined.
- Assigned this variable in SceneBuilder by setting the fx:id for the textfield to "num_dose_1"
- Added to the function "getResults" to print the text in the "num_dose_1" textfield to the console on the button click.
- Lengthened the following button so that text is not cut-off.
![image](https://user-images.githubusercontent.com/55509733/105085832-4ffbd400-5a66-11eb-8b2f-71f0c4f013b2.png)


How to test:
- Validate that all existing functionality still works as expected
- Type anything into the first textfield, see image below
![image](https://user-images.githubusercontent.com/55509733/105062485-a3f7c000-5a48-11eb-80f2-cdd8ab955b4e.png)
- Click "See Results" button
- Confirm that button takes you to "results.fxml" page, and that textfield input is printed to the console.

Let me know if there are any questions, thanks!
